lang/python34/Portfile

```diff
@@ -9,7 +9,7 @@ name                python34
 epoch               20170810
 # Remember to keep py34-tkinter and py34-gdbm's versions sync'd with this
 version             3.4.10
-revision            4
+revision            5
```

Comment:
Don't change the revision since the issue is that the port previously couldn't be built, so there's no benefit to making anyone reinstall if it did build for them.

Reply:
Ah, ok. I figured I'd bump it as this just _might_ affect other targets. Fixed.